ccd-definition/ComplexTypes/Party.json

```diff
@@ -109,5 +109,14 @@
     "ElementLabel": "Party name",
     "FieldShowCondition": "type = \"DO_NOT_SHOW\"",
     "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Party",
+    "ListElementCode": "typeDisplayValue",
```

Comment:
@pd-dudkiewicz - as discussed, please can we change this to partyTypeDisplayValue?

Reply:
done